shawerma/tests.py

```diff
@@ -0,0 +1,108 @@
+import datetime
+
+from django.db.models.functions import TruncMonth
+from django.conf import settings
+from django.utils import timezone
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.db.models import Max,Count,Avg,Sum,F
+
+from rest_framework.test import APIRequestFactory,RequestsClient,APIClient
+from rest_framework import status
+
+from shawerma.models import Order,User,MenuItem
+
+
+class ViewTestCase(TestCase):
+    fixtures = ['fixtures/test.json']
+
+    def test_make_order(self):
+        client = APIClient()
+        #u = User.objects.filter(username="Ali")
+        client.login(username="admin", password="password123")
+        query1=MenuItem.objects.filter(id = 2)
+        date = datetime.datetime.now().time()
+        response = client.post('/shawerma_order/create/',{'address':'Ram'},format='json')
+        self.assertEqual (response.status_code, 201)
+        client.logout()
```

Comment:
please validate that the order is created... you are just checking the status code only without checking what the API is doing